wpengine-phpcompat.php

```diff
@@ -73,21 +73,23 @@ public static function init() {
 	 * @return null
 	 */
 	function start_test() {
-		global $wpdb;
+		if ( current_user_can( 'edit_posts' ) ) {
```

Comment:
The menu registration call uses `'administrator'` (which is a deprecated capability). I'd recommend using `'manage_options'`